yarn-project/pxe/src/kernel_prover/hints/build_private_kernel_reset_private_inputs.ts

```diff
@@ -454,7 +454,17 @@ export class PrivateKernelResetPrivateInputsBuilder {
 
     const numLogs = this.previousKernel.end.encryptedLogsHashes.filter(l => !l.logHash.randomness.isZero()).length;
     const numToSilo = Math.max(0, numLogs - this.numTransientData);
-    this.requestedDimensions.ENCRYPTED_LOG_SILOING_AMOUNT = numToSilo;
+    // The reset circuit checks that capped_size must be greater than or equal to all non-empty logs.
+    // Since there is no current config with ENCRYPTED_LOG_SILOING_AMOUNT = 0 (only 1+), it defaults to 1,
+    // so the circuit fails when we have more than 1 log and require no siloing.
+    const numLogsNoSiloing = this.previousKernel.end.encryptedLogsHashes.filter(
+      l => !l.logHash.isEmpty() && l.logHash.randomness.isZero(),
+    ).length;
+    const cappedSize = !numToSilo && numLogsNoSiloing > 1 ? numLogsNoSiloing : numToSilo;
+    // NB: This is a little flimsy, and only works because we have either ENCRYPTED_LOG_SILOING_AMOUNT=1 or 8.
+    // e.g. if we have 2 logs that need siloing, and 2 that dont, then numLogs = ENCRYPTED_LOG_SILOING_AMOUNT = 2
+    // This would fail because the circuit thinks that cappedSize = 2, but we have 4 logs.
+    this.requestedDimensions.ENCRYPTED_LOG_SILOING_AMOUNT = cappedSize;
```

Comment:
This fixes something I believe is in master - reset will fail for any number of encrypted logs which don't need siloing above 1. 
It works now because we'd never tested more than 1 encrypted log with randomness set to 0. In that case, `ENCRYPTED_LOG_SILOING_AMOUNT=0` in the above function, and the reset variant with `=1` is chosen. Since there is only 1 log overall, it passes. Otherwise, it fails the `capped_size` check for the same reasons we modify the `NULLIFIER_SILOING_AMOUNT` just above this fn.